libs/build-all.sh

```diff
@@ -15,6 +15,12 @@ NSS_SHA256="fb2d54d507ceb185bac73f492cce7086a462d41977c2378aba9dd10e04448cf3"
 
 # End of configuration.
 
+if [ "${PWD##*/}" != "libs" ]
```

Comment:
IIUC this means the script can only be executed if the cwd is a folder named `libs`. Alternatively you could have done `if [ ! -f "$(pwd)/build-all.sh" ];`